Code-Hills/codehills-be
# Removed role restriction on rating field and categories

# What does this PR do?

# Description of Task to be completed?

## Requirements


- [ ] Require dependences installation
- [ ] Require .env update

# How should this be manually tested?
Just login as normal and access ratingCategories and ratingFields routes with any role. you should be able to view them
# Any background context you want to provide?

# What are the relevant pivotal tracker stories?

# Screenshots (if appropriate)

# Questions:
- N/A
